x/emissions/keeper/msgserver/msg_server_demand.go

```diff
@@ -2,13 +2,21 @@ package msgserver
 
 import (
 	"context"
+	"time"
+
+	"github.com/allora-network/allora-chain/x/emissions/metrics"
 
 	"github.com/allora-network/allora-chain/x/emissions/types"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
-func (ms msgServer) FundTopic(ctx context.Context, msg *types.FundTopicRequest) (*types.FundTopicResponse, error) {
+func (ms msgServer) FundTopic(ctx context.Context, msg *types.FundTopicRequest,
+) (
```

Comment:
I know this is a nitpick, but please don't format things like this.  Either

`func foobar(a string, b int) (float, bool)`
or
```
func foobar(
    a string,
    b int,
) (float, bool)
```